src/Dmux/workflow/ngs_qaqc/fastq.smk

```diff
@@ -42,7 +41,6 @@ rule fastq_screen:
         subset              = 1000000,
         aligner             = "bowtie2",
         output_dir          = lambda w: config['out_to'] + "/" + w.project + "/" + config['run_ids'] + "/" + w.sid + "/fastq_screen/",
-    # container: "docker://rroutsong/dmux_ngsqc:0.0.1",
     containerized: "/data/OpenOmics/SIFs/dmux_ngsqc_0.0.1.sif"
```

Comment:
We need to add an option to point to a sif cache and dynamically resolve one of the following: a local SIF on the file-system or a URI to pull an image from Dockerhub.

Reply:
I have a solution to this issue in the next coming PR. I have serialized the server-centric SIF directories and dynamically adding the specific server configuration at initialization time. 

Ends up like:

`containerized: server_config["sif"] + "dmux_ngsqc_0.0.1.sif"`

SIF cache is always specified at execution time through environmental variables and subprocess.